modules/apps/dispatch/dispatch-lang/src/main/resources/content/Language.properties

```diff
@@ -0,0 +1,27 @@
+add-scheduled-task=Add Scheduled Task
+add-trigger=Add Trigger
+an-error-occurred-while-processing-the-scheduled-task=An error occurred while processing the scheduled task.
+are-you-sure-you-want-to-delete-the-selected-logs=Are you sure you want to delete the selected logs?
+are-you-sure-you-want-to-delete-the-selected-trigger=Are you sure you want to delete selected trigger?
+cron-expression=Cron Expression
+dispatch=Dispatch
+dispatch-configuration-name=Dispatch Configuration Name
+dispatch-trigger=Dispatch Trigger
+edit-trigger=Edit Trigger
+execute-once=Execute Once
+last-fire-time=Last Run Time
```

Comment:
@igorbeslic looks like some of the existing keys carried over don't have matching keys and values, I think we should fix them.
`last-run-time=Last Run Time`